examples/step-63/step-63.cc

```diff
@@ -630,6 +692,11 @@ namespace Step63
     right_hand_side.value_list(scratch_data.fe_values.get_quadrature_points(),
                                rhs_values);
 
+    // If we are using streamline diffusion we must add its contribution
+    // to both the cell matrix and the cell right-handside. If we are not
+    // using streamline diffusion, setting $\delta=0$ negates this contribution
+    // below and we are left with the standard, Galerkin finite element
+    // assembly.
     const double delta = settings.with_streamline_diffusion ?
```

Comment:
I changed up this part a little. If we are setting `delta=0` for the "without_streamline" case, to me it makes sense to not have another `if(with_streamline)` when we assemble the `cell_matrix(i,j)+=...`, since if `delta=0` the streamline diffusion contribution will be zero. Plus, the "without_streamline" case is only used to show a picture example in the **Results** section, and all iteration counts are based on computations with_streamline, so this is another reason to me it makes sense to combine every term into one `cell_matrix(i,j)+=...`

Let me know if you have any thoughts on this.